components/Discussion/graphql/store.js

```diff
@@ -84,6 +86,28 @@ export const bumpCounts = ({ comment, initialParentId }) => draft => {
   }
 }
 
+/**
+ * Give a new comment, bump the tag buckets counts
+ */
+export const bumpTagCounts = ({ comment, initialParentId }) => draft => {
+  const nodes = draft.discussion.comments.nodes
+
+  const rootComment = comment.parentIds?.length
+    ? nodes.find(n => n.id === comment.parentIds[0])
+    : comment
+
+  const tags = rootComment.tags
+
+  if (!tags?.length) return
+
+  for (const tag of tags) {
+    const bucket = draft.discussion.tagBuckets?.find(b => b.value === tag)
+    if (bucket) {
+      bucket.count++
+    }
+  }
+}
```

Comment:
`parentIds` is always present. We also relay on it in `Comments.js`.

```suggestion
  const parentId = comment.parentIds[0]
  const rootComment = parentId
    ? nodes.find(n => n.id === parentId)
    : comment

  const affectedTags = rootComment?.tags

  if (!affectedTags?.length) return

  for (const tag of affectedTags) {
    const bucket = draft.discussion.tagBuckets?.find(b => b.value === tag)
    if (bucket) {
      bucket.count =+ 1
    }
  }
}
```